app/javascript/utils/form/hoc.js

```diff
@@ -1,7 +1,12 @@
+// This files contains some High Level Components for helping building forms.
```

Comment:
just googled HOC to find out what it is, maybe instead of saying High Level Components you can say "Higher-Order Components (HOC)" in this comment to help clarify